examples/basic.ipynb

```diff
@@ -40,7 +40,7 @@
    "id": "comfortable-insurance",
    "metadata": {},
    "source": [
-    "First let's prompt for your CMR credentials (UAT)."
+    "First let's prompt for your CMR credentials (UAT). Your credentials are stored without needing to hit enter in either field."
```

Comment:
I could be dense, but I _still_ found this confusing; I just really want to click something after typing in username / password.  Maybe text like "Type your credentials in the text boxes below and then execute the next cell."

Reply:
I agree. I don't like it, but was low on my priority list. Is there a quick fix I could apply?

Reply:
I'll look at this the next time I revisit the notebook.